requirements.txt

```diff
@@ -4,7 +4,7 @@ pipenv
 toml
 delegator.py
 pyyaml
-gitpython==2.1.8
+gitpython==3.1.0
```

Comment:
Please match the version specifier so we run the same version in the cluster as locally.